src/endpoints/pcm.js

```diff
@@ -0,0 +1,13 @@
+import CRUDExtend from '../extends/crud'
+
+class PCMEndpoint extends CRUDExtend {
```

Comment:
We're still uncertain of the name "pcm" as it may change. As anyone can see the source, we should be as descriptive as possible with the name. Let's follow-up with David to confirm if this acronym can still be used for now but in the future I imagine pcm endpoints will replace our current "products" endpoints and that will be a breaking change in the SDK.